### PR TITLE
[Magma] [FAQ] Updating magma Git FAQ

### PR DESCRIPTION
**Pull request by saurabhsoni88**
Signed-off-by: Saurabh Soni <sscloud@fb.com>

## Summary

Adding additional FAQ questions.

## Test Plan

    Added new points in same format.

### CODE REVIEW

File: docs/readmes/faq/magma_faq.md [hunk 1]
```diff
@@ -65,6 +65,22 @@ This section lists some of the commonly asked questions related to Magma operati
     - From the popup window, select tab **COMMANDS**. Then, check section **Reboot**.
     - Click on **Reboot** to reboot the AGW node.
 
+### Does AGW supports IPv6?
+  - IPv6 and dual stack IPv4V6 session support are currently under active development and we are working towards making this feature ready as part of release 1.4.
+
+### How to change hardware UUID of AGW?
+  - Hardware UUID is in `/etc/snowflake`.
+  - Delete file `/etc/snowflake` and restart magma@magmad systemd service, which will create new one.
```
Comment (uri200) [inline docs/readmes/faq/magma_faq.md:73]: Maybe add that if you do that you will need to modify the id through swagger API so the gateway is registered properly on orcr8

`/networks/{network_id}/gateways/{gateway_id}/device`

File: docs/readmes/faq/magma_faq.md [hunk 2]
```diff
@@ -65,6 +65,22 @@ This section lists some of the commonly asked questions related to Magma operati
...
+  - **NOTE** If you reinstall magma, the UUID hardware ID will change.
+
+### How to switch between info/debug mode in mme logs?
+  - Modify `/var/opt/magma/configs/mme.yml` and add the line `"log_level: DEBUG"`,
+  - Restart the mme service with `sudo service magma@mme restart`.
+  - **NOTE** Expect impact of service while doing above procedure.
```
Comment (uri200) [inline docs/readmes/faq/magma_faq.md:79]: should we mention that this can be done through orc8r too?

File: docs/readmes/faq/magma_faq.md [hunk 3]
```diff
@@ -84,6 +100,10 @@ This section lists some of the commonly asked questions related to Magma operati
   - Click on **Try it out** button on right hand side.
   - Put in the required inputs and click **Execute**.
 
+### How to verify which services are running in Orc8r-controller?
+  - **Get the controller pods:** `kubectl -n magma get pods`.
+  - **Execute the command in the pod:** `kubectl exec -n magma <CONTROLLER PODNAME> supervisorctl status`.
+
```
Comment (uri200) [inline docs/readmes/faq/magma_faq.md:111]: i may be using old commands, what is that `magma` keyword on kubectl?

Comment (saurabhsoni88) [reply docs/readmes/faq/magma_faq.md:106]: Will modify it to "kubectl exec -n <NAME SPACE> orc8r-controller-6b85fbdf77-76mmj -- supervisorctl status". This we checked, as per latest format.

Comment (saurabhsoni88) [reply docs/readmes/faq/magma_faq.md:111]: Will modify it to "kubectl exec -n NAME-SPACE orc8r-controller-6b85fbdf77-76mmj -- supervisorctl status. This we checked, is as per latest format.

File: docs/readmes/faq/magma_faq.md (hunk 1 above)
Comment (saurabhsoni88) [reply docs/readmes/faq/magma_faq.md:73]: Yes, let's do that.

File: docs/readmes/faq/magma_faq.md (hunk 2 above)
Comment (saurabhsoni88) [reply docs/readmes/faq/magma_faq.md:79]: Mostly they try with AGW and operationally AGW is mostly active, so let's keep it now from AGW.

**Review by uri200 — APPROVED** [review]

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

**Pull request merged**